spec/factories/presentations.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :presentation do
+    title "MyString"
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.